packages/core/content-releases/server/src/utils/index.ts

```diff
@@ -1,9 +1,19 @@
 import type { UID, Data, Core } from '@strapi/types';
+import type { SettingsService } from '../services/settings';
 
-export const getService = (
-  name: 'release' | 'release-validation' | 'scheduling' | 'release-action' | 'event-manager',
+type Services = {
+  release: any;
+  'release-validation': any;
+  scheduling: any;
+  'release-action': any;
+  'event-manager': any;
+  settings: SettingsService;
+};
```

Comment:
@Feranchz I updated the `getService` method to be typed, but we might as well want to add the rest of services type. Do you prefer to fill them in another PR?